modules/microshift-creating-ostree-iso.adoc

```diff
@@ -0,0 +1,139 @@
+>> End example output
+----
+
+. Download the container image using the ID and get the image ready for use by running the following commands:
++
+[source,terminal]
+----
+# sudo composer-cli compose image ${BUILDID}
+sudo chown $(whoami). ${BUILDID}-container.tar
+sudo chmod a+r ${BUILDID}-container.tar
+----
+//Q: Can we break these into three explained commands in series in this step? It's a general rule; helps support know where something may have gone wrong
+
+. Bootstrap a server on port 8085 for the `ostree` container image to be consumed by the ISO by running the following command:
++
+[source,terminal]
+----
+IMAGEID=$(cat < "./${BUILDID}-container.tar" | sudo podman load | grep -o -P '(?<=sha256[@:])[a-z0-9]*')
+sudo podman run -d --name=minimal-microshift-server -p 8085:8080 ${IMAGEID}
+----
++
+This command also returns the ID of the container saved in the `IMAGEID` variable for monitoring.
+
+. Generate the installer blueprint file by running the following command:
++
+[source,terminal]
+----
+# cat > microshift-installer.toml <<EOF
+name = "microshift-installer"
+
+description = ""
+version = "0.0.0"
+modules = []
+groups = []
+packages = []
+EOF
+----
+
+. Add the blueprint to the Image Builder by running the following command:
++
+[source,terminal]
++
+----
+# sudo composer-cli blueprints push microshift-installer.toml
+----
+
+. Start the `ostree` ISO build by running the following command:
++
+[source,terminal]
+----
+BUILDID=$(sudo composer-cli compose start-ostree --url http://localhost:8085/repo/ --ref "rhel/9/$(uname -m)/edge" microshift-installer edge-installer | awk '{print $2}')
+----
++
+This command also returns the identification (ID) of the build for monitoring.
+
+. You can check the status of the build periodically by running the following command:
++
+[source,terminal]
+----
+# sudo composer-cli compose status
+----
++
+.Example output for a running build
++
+[source,terminal]
+----
+ID                                     Status     Time                      Blueprint              Version   Type               Size
+c793c24f-ca2c-4c79-b5b7-ba36f5078e8d   RUNNING    Wed Jun 7 13:22:20 2023   microshift-installer   0.0.0     edge-installer
+>>
+----
++
+.Example output for a completed build
++
+[source,terminal]
+----
+ID                                     Status     Time                      Blueprint              Version   Type               Size
+c793c24f-ca2c-4c79-b5b7-ba36f5078e8d   FINISHED   Wed Jun 7 13:34:49 2023   microshift-installer   0.0.0     edge-installer
+>>
+----
+
+. Download the ISO using the ID and get the ISO ready for use by running the following commands
++
+[source,terminal]
+----
+# sudo composer-cli compose image ${BUILDID}
+sudo chown $(whoami). ${BUILDID}-installer.iso
+sudo chmod a+r ${BUILDID}-installer.iso
+----
+//Q: Can we break these into three explained commands in series in this step? It's a general rule; helps support know where something may have gone wrong
```

Comment:
Yes, we can split the last 2 commands into a separate block if you like, stating that they ensure that the downloaded ISO file is owned and readable by the current user.